README.rst

```diff
@@ -206,6 +206,8 @@ enabled by adding 'python_venv' to the list of enabled extensions.
         // The python executable to use in the venv.
         "python": "python2.7",
         // Names of requirements files to install in the venv.
+        "require_setup_py": true,
+        // Optional flag to enable building an rpm with, without a setup.py file. Default is true if not present.
```

Comment:
It looks like the comments for `require_setup_py` and `requirements` are swapped. This line should go above the new option.